creme/activities/bricks.py

```diff
@@ -325,6 +326,32 @@ def detailview_display(self, context):
         ))
 
 
+class ActivityCalendarBrick(Brick):
+    id = QuerysetBrick.generate_id('activities', 'activity-calendar')
+    verbose_name = 'Activities calendar'
+    dependencies = (Activity, Contact, Calendar,)
+    template_name = 'activities/bricks/activity-calendar.html'
+    permissions = 'activities'
+
+    def detailview_display(self, context):
+        user = context['user']
+        settings = {
+            "utc_offset": get_current_utc_offset(),
+            **CalendarConfigItem.objects.for_user(user).as_dict(),
+        }
+
+        return self._render(self.get_template_context(
+            context,
+            calendar_sources=list(
+                Calendar.objects.filter(
+                    user=context['user'],
+                    is_default=True,
+                ).values_list('pk', flat=True)
+            ),
+            calendar_settings=settings,
```

Comment:
If I understand, the brick does not use the current entity (i.e. `context['object']` ) (I look at the HTML/JS code too). So it behaves like a portal brick, but it's displayed only on detail views (displaying the same activities) hum...
Should we filter the displayed activities?
- Any relation types between the Activities & the current instance?
- Something smarter in some cases (e.g. for an Organisation we use also the related Contacts -- see FutureActivitiesBrick)?